src/commands/Games/higherLower.ts

```diff
@@ -194,10 +209,23 @@ export default class extends SkyraCommand {
 		}
 
 		// Say bye!
-		await game.gameMessage.edit(new MessageEmbed()
-			.setColor(Slotmachine.FAIL_COLOUR)
-			.setTitle(message.language.tget('COMMAND_HIGHERLOWER_CANCEL_TITLE'))
-			.setDescription(message.language.tget('COMMAND_HIGHERLOWER_CANCEL_DESCRIPTION', message.author.username)));
+		const { TITLE, DESCRIPTION } = message.language.tget('COMMAND_HIGHERLOWER_CANCEL');
+		await game.gameMessage.edit({
+			content: null,
+			embed: new MessageEmbed()
+				.setColor(getColor(message))
+				.setTitle(TITLE)
+				.setDescription(DESCRIPTION(message.author.username))
+		});
+	}
+
+	private async reactionCollector(ctx: EmojiPromiseContext, message: KlasaMessage, game: HigherLowerGameData, reaction: LLRCData) {
+		// Ignore if resolve is not ready
+		if (typeof ctx.resolve !== 'function'
+						// Run the collector inhibitor
+						|| await this.reactionInhibitor(message, game.gameMessage!, reaction)) return;
+		this.client.clearTimeout(ctx.timeout!);
+		ctx.resolve(reaction.emoji.name as ReactionEmoji);
```

Comment:
Pretty sure that you can use the `isThenable` function from `@klasa/utils` (already available in Skyra) to validate that `ctx` is a promise but I never used the function myself. Otherwise you can use `isfunction` from the same util lib here for sure.